src/RegistryCoordinator.sol

```diff
@@ -423,6 +424,16 @@ contract RegistryCoordinator is
         _setEjector(_ejector);
     }
 
+    /**
+     * @notice Sets the reregistration delay, which is the time an operator must wait after 
+     * ejection before registering for any quorum
+     * @param _reregistrationDelay the new reregistration delay
+     * @dev only callable by the owner
+     */
+    function setReregistrationDelay(uint256 _reregistrationDelay) external onlyOwner {
+        reregistrationDelay = _reregistrationDelay;
```

Comment:
might want to make the name of this reflect the fact that it's only used when ejection occurs?
e.g. the 'reregistration delay' (current name) is not enforced when an operator is 'churned out'

as a side note, I think this may present a minor "attack" whereby an operator de-registers themself instead of getting ejected, to evade this delay?
You could address this by making `ejectOperator` always set the `lastEjectionTimestamp` and no longer revert if the operator is not registered.

Reply:
suggested new names:
ejectionLength
ejectionReregistrationDelay or ejectionReregistrationPeriod
ejectionCooldownPeriod

I don't particularly like any of these but these are some ideas